app/connectors/Sub09Connector.scala

```diff
@@ -77,10 +77,9 @@ class Sub09Connector @Inject()(appConfig: AppConfig,
 
     metricsReporter.withResponseTimeLogging("mdg.get.company-information") {
       http.GET[Option[MdgSub09CompanyInformationResponse]](uri, headers = headers)
-        .map(_.map(v => CompanyInformation(v.name, v.address))).recover {
+        .map(_.map(v => CompanyInformation(v.name, v.consent.get, v.address))).recover {
```

Comment:
As the consent field is optional it's better that we do a getOrElse otherwise if it's None we'll get a NoSuchElementException.